### DISCUSSION TITLE
TG Token Bot 設定問題

### DISCUSSION THREAD

**Discussion by simonlkch** [Baiyuetribe/kamiFaka#66] [Bug反馈]
在 Tg机器人参数设置 畫面

填寫TG_Token 時出現 请填写正确的 TG TOKEN

我的TG_Token 在 botfather 中取出來的, 在其他地方是可以用的, 是出現什麼問題?

![image](https://user-images.githubusercontent.com/3492546/116870900-f484d900-ac45-11eb-9b86-345112463c40.png)

**Comment by Baiyuetribe** [comment]
TG发卡在v1.4之后移除了，使用者极少。若要使用，请使用上古版本
